docs/dev/code_reference/nist.md

```diff
@@ -0,0 +1,17 @@
+# NIST URLs
+
+::: netutils.nist
+    options:
+        show_submodules: True
+
+**Notes:** 
+<br/>
+In order to use the URLs generated by netutils.nist.get_nist_urls*, you will need an api key provided by NIST [here]('https://nvd.nist.gov/developers/request-an-api-key').  This key will need to be passed in as an additional header in your request in the form of `{"apiKey": "<key_value>"}` as stated by NIST in their [Getting Started]('https://nvd.nist.gov/developers/start-here') section.
+<br/><br/>
+This module currently only supports the following OS platforms:
```

Comment:
Add a new line before and after here.

Reply:
I think this should also be resolved/outdated.  Requested changes made.